cmd/importverifier/importverifier.go

```diff
@@ -276,3 +291,47 @@ func logForbiddenPackages(base string, forbidden []string) {
 		log.Printf("--- %s\n", forbiddenPackage)
 	}
 }
+
+// Check if any imports are disallowed
+// Exec e.g find . -name "*.go" |  grep -v vendor/ | grep -v cmd/kubeadm/app | { xargs grep 'github.com/pkg/errors' -sl || true; }
+func verifyDisallowedImports() error {
+
+	// Manually create a single entry for github/pkg/errors
+	disallowedImports := []DisallowedImport{{
+		Pkg:          "github.com/pkg/errors",
+		IgnoredTrees: []string{"cmd/kubeadm/app", "cmd/importverifier"},
```

Comment:
i think ideally, instead of hardcoding paths in the tool source code this should be on the side of the wrapping bash scripts - i.e. passed as flags.